src/platform/forms-system/src/js/components/ConfirmationView/components.jsx

```diff
@@ -254,14 +259,15 @@ export const SavePdfDownload = ({ pdfUrl, trackingPrefix, className }) => {
         filetype="PDF"
         href={pdfUrl}
         onClick={onClick}
-        text="Download a copy of your VA Form"
+        text={`Download a copy of your VA Form ${formId}`}
       />
     </div>
   ) : null;
 };
 
 SavePdfDownload.propTypes = {
   className: PropTypes.string,
+  formId: PropTypes.string,
```

Comment:
Can you also add to `SavePdfDownloadWithContext` below? That will help with cases where users are manually using <ConfirmationView.SavePdfDownload>

Reply:
Great catch, updated! :+1: